jenkins/Jenkinsfile-blossom.premerge-databricks

```diff
@@ -129,7 +129,7 @@ String getDbType() {
 void databricksBuild() {
     def CLUSTER_ID = ''
     def SPARK_MAJOR = BASE_SPARK_VERSION_TO_INSTALL_DATABRICKS_JARS.replace('.', '')
-    def dbfs_path = "$INIT_SCRIPTS_DIR-$DB_TYPE"
+    def ws_path = "$INIT_SCRIPTS_DIR-$DB_TYPE"
```

Comment:
another way of doing this is to add spark-rapids github repo under /Repos. Then we can reference an initscript path /Repos/user@email/spark-rapids/...

Reply:
If we create a mirror spark-rapids GitHub repo on Databricks /Repos/user@email/spark-rapids dir, it is inflexible to point the init_script files at the PR for the pre-merge pipeline, or pointing to someone's private GitHub repo for testing pipelines?